Add files via upload

konopkagrzegorz opened the pull request:
Added Gdynia faculty of WSB University in Gdańsk due to different email domain.

philipto commented:
@konopkagrzegorz wsb/wsb-gdynia.txt: this request has an incorrect file path format, please ensure that you are familiar with     this repository rules located at the bottom of [this page](https://github.com/JetBrains/swot) and change     your request accordingly. If you don't know how to change request, close it and open a new one.     If you are not able to make the pull request according the instructions, please send us an email to support@jetbrains.com. Please mention this request URL in the message.

konopkagrzegorz commented:
@philipto i deleted wrong file with inncorect path and i added new - according to guidelines. If I did something wrong please inform me. I will close pull request and create a new one.

philipto commented:
@konopkagrzegorz I have to close this request without merging, because you requested to add a non-existent domain to this repository: `Host student.wsb.gdynia.pl not found: 3(NXDOMAIN)` Please ensure that you are familiar with this repository rules located at the bottom of [this page](https://github.com/JetBrains/swot) and submit a new request with the correct information. Thank you for understanding!
